packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IForegroundColorTokens, FontTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { View, ViewProps } from 'react-native';
+
+export const tabsName = 'Tabs';
+
+export interface TabsContextData {
+  /*
+   ** The currently selected TabsItem's key
+   */
+  selectedKey: string | null;
+
+  /*
+   ** Index of currently selected key
+   */
+  getTabId?: (key: string, index: number) => string | null;
+
+  /*
+   ** Updates the selected tabsItem and calls the client’s onTabsClick callback
+   */
+  onTabsClick?: (key: string) => void;
+
+  /*
+   ** Updates the selected tabsItem's ref to set as the default tabbable element
+   */
+  updateSelectedTabsItemRef?: (ref: React.RefObject<any>) => void;
+
+  /*
+   ** Array of tabsItem keys in the group
+   */
+  tabsItemKeys?: string[];
+
+  /*
+   ** A Map to for a TabItems corresponding view
+   */
+  views?: Map<string, React.ReactNode[]> | null;
```

Comment:
Tabs is what knows what the currently selected tabs item is, so by doing something like views.get(selectedTabItem) in tabs we're able to easily get and render the selected view. If the views where held by tabs item it would be difficult to propagate that data from tabs item to tabs.

Reply:
The views map holds the children of the TabsItem by their key. This views is held on the context so it can be accessed and rendered under Tabs.